welcome.md

```diff
@@ -4,7 +4,7 @@ sd_hide_title: true
 
 # Home
 
-<center><img src="https://github.com/pymc-devs/brand/blob/main/logos/1-pymcdevs.png?raw=true" width="75%"/></center>
+<center><img src="https://github.com/pymc-devs/brand/blob/main/pymc/pymc_logos/1-pymcdevs.png" width="75%"/></center>
```

Comment:
This links to the new location but doesn't link to the raw image, so the image is still broken in both PR preview and pymc.ip